kzg_prover/benches/kzg.rs

```diff
@@ -261,75 +260,65 @@ fn bench_kzg<
 }
 
 fn criterion_benchmark(_c: &mut Criterion) {
-    const N_CURRENCIES: usize = 2;
-    const N_POINTS: usize = 3;
+    const N_CURRENCIES: usize = 1;
+    const N_POINTS: usize = 2;
 
     // Demonstrating that a higher value of K has a more significant impact on benchmark performance than the number of users
     #[cfg(not(feature = "no_range_check"))]
     {
         const K: u32 = 18;
-        const N_USERS: usize = 16;
+        const N_USERS: usize = (1 << 17) + (1 << 16) - 6;
```

Comment:
Can you try to replace it with 2^K-6?

Reply:
Indeed, that is worked. Similarly, we can apply this to `N_POINTS` as follows: 
```Rust
const N_POINTS = N_CURRENCIES + 1;
```